contracts/experimental/ERC20ProxyForZRC2/tasks/deployProxy.ts

```diff
@@ -0,0 +1,37 @@
+task("deployProxy", "deploy an ERC20 proxy for a ZRC2 contract")
+  .addPositionalParam("zrc2Address")
+  .setAction(async ({zrc2Address}) => {
+    console.log(`Deploying an ERC20 proxy for the ZRC-2 at ${zrc2Address}`);
+    // ZRC2 addresses frequently have bad checksums, hence toLowerCase().
+    const contract = await ethers.deployContract("ZRC2ERC20Proxy",
+                                                 [zrc2Address.toLowerCase()],
+                                                 { gasLimit: 1_000_000 });
```

Comment:
Is setting the `gasLimit` necessary? (i.e. does estimate gas fail/lie for these transactions?)

Setting a fixed limit means this can break if gas charges change in the future.

Reply:
Well caught - sorry again - inherited from previous script for no reason I could see. Will fix in other PR ..